### DISCUSSION TITLE
Space Teams not letting me access anything but Demos

### DISCUSSION THREAD

**Discussion by ashritadara** [SimDynamX/SpaceTeamsPRO#58] [2024 Lunar SAR Competition Q&A] [answered]
I've been trying to access the other simulations and it is not letting me. and when i enter the demo it is not letting me scroll through to see the other planets or earth or anything. when i login it said "Personal Use (UNPAID)". How do i change that??

**Comment by connorjak** [comment, marked as answer]
The fix for this is accepting the free trial license on spaceteamspro.com after making an account
